pkg/uploader/uploader_test.go

```diff
@@ -100,7 +100,7 @@ func TestUploaderSuite(t *testing.T) {
 }
 
 func (suite *UploaderSuite) TestUploaderExceedsFileSizeLimit() {
-	_, err := uploader.NewUploader(suite.DB(), suite.logger, suite.storer, 361*uploader.MB)
+	_, err := uploader.NewUploader(suite.DB(), suite.logger, suite.storer, 251*uploader.MB)
```

Comment:
note to self: maybe this should be:
```suggestion
	_, err := uploader.NewUploader(suite.DB(), suite.logger, suite.storer, MaxFileSizeLimit + (1 * uploader.MB))
```